lib/libzfs/libzfs_util.c

```diff
@@ -1940,3 +1942,61 @@ zprop_iter(zprop_func func, void *cb, boolean_t show_all, boolean_t ordered,
 {
 	return (zprop_iter_common(func, cb, show_all, ordered, type));
 }
+
+/*
+ * Fill given version buffer with zfs userland version
+ */
+int
+zfs_version_userland(char *version, int len)
+{
+	(void) strlcpy(version, ZFS_META_GITREV, len);
+
+	return (1);
```

Comment:
might as well make it return (void) since it only ever returns 1, and the caller ignores it.

Reply:
For whatever reason, I was just making both function prototypes the same.  I will change it to (void).